.github/workflows/validate_json.yml

```diff
@@ -1,5 +1,8 @@
 name: Validate JSON Schema files
-on: [push]
+on:
+  push:
+    paths:
+      - "v*.*/*.json"
```

Comment:
after reading  @Alessandro100 comment on my other PR, I figured that this here would now also need a package vsion check like here: https://github.com/MobilityData/gbfs-json-schema/blob/747c717249bdc4b20ca8c2e9011e8347da0ba8f5/.github/workflows/typescript_pr_check.yml#L18 . Wdyt?